pyproject.toml

```diff
@@ -1,9 +1,11 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools",
     "wheel",
-    "ninja; sys_platform != 'win32'",
-    "cmake>=3.14",
-    "pybind11~=2.6.1",
+    # "scikit-build>=0.13",
+    # test fix
+    "scikit-build @ git+https://github.com/scikit-build/scikit-build@c18845a1c8a0af9fd9f04a531b501a17650bfa98",
+    "ninja",
+    "cmake>=3.16",
+    "pybind11>=2.8.1"
 ]
-build-backend = "setuptools.build_meta"
```

Comment:
Ignore our recommendation! You still should have this! I'll fix the docs right now.